source/server-discovery-and-monitoring/tests/README.rst

```diff
@@ -56,7 +56,7 @@ so far. It has the following keys:
 A "server" object represents a correct ServerDescription within the client's
 current TopologyDescription. It has the following keys:
 
-- type: A ServerType name, like "RSSecondary".
+- type: A ServerType name, like "RSSecondary". For async and multi-threaded implementations, the "PossiblePrimary" type should be interpreted as "RSUnknown". See `ServerType <../server-discovery-and-monitoring.rst#servertype>`_ for more details.
```

Comment:
Suggestion per @p-mongo's comment in https://github.com/mongodb/specifications/pull/683#issuecomment-550047807.

```suggestion
- type: A ServerType name, like "RSSecondary". See `ServerType <../server-discovery-and-monitoring.rst#servertype>`_ for details pertaining to async and multi-threaded drivers.
```

I realize this PR originally used "RSUnknown", but I only see "Unknown" used in the spec itself. I'm not sure if that changed in the time since this PR was created, but if so it's probably all the more reason to refer back to the spec instead of duplicating spec language in the test README.

Reply:
Thanks, I like this change. This should be ready to go now. Better late than never!